rules/alerting_test.go

```diff
@@ -684,7 +684,7 @@ func TestSendAlertsDontAffectActiveAlerts(t *testing.T) {
 	// Set an active alert.
 	lbls := labels.FromStrings("a1", "1")
 	h := lbls.Hash()
-	al := &Alert{State: StateFiring, Labels: lbls, ActiveAt: time.Now()}
+	al := &Alert{State: StateFiring, Labels: lbls, Annotations: labels.EmptyLabels(), ActiveAt: time.Now()}
```

Comment:
Note to reviewers: this change is just because `Alert.deepCopy()` also copies annotations. If original annotations are nil, the `labels.Label.Copy()` returns an empty slice and the deep equal assertion done below doesn't match. However, an empty slice and `nil` are logically the same, so shouldn't be a regression.